src/foam/dao/FixedSizeDAO.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Not copyright Google.

/**
 * @license
 * Copyright 2018 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */